packages/plugin-conversation/src/decryption-transforms.js

```diff
@@ -37,8 +37,15 @@ export const transforms = toArray(`inbound`, {
     }
 
     const usableKey = conversation.encryptionKeyUrl || key;
+    const decryptionFailureMessage = ctx.spark.conversation.config.decryptionFailureMessage;
+
     if (usableKey) {
-      promises.push(ctx.transform(`decryptPropDisplayName`, usableKey, conversation));
+      promises.push(ctx.transform(`decryptPropDisplayName`, usableKey, conversation)
+        .catch((error) => {
+          ctx.spark.logger.error(`plugin-conversation: failed to handle decryption `, error.stack || error.toString());
```

Comment:
The logs would show the following message on failure:
plugin-conversation: failed to handle decryption  KmsError: User 40b446fe-175c-4628-8a9d-fd782f47ebe6 doesn't have authorization to access resource https://encryption-a.wbx2.com/encryption/api/v1/resources/d31a41dc-47d6-43bd-80d5-ff35a39dc8ad